ui/components/app/multilayer-fee-message/multi-layer-fee-message.test.js

```diff
@@ -42,5 +66,29 @@ describe('Multi layer fee message', () => {
     expect(getByText('Layer 1 fees')).toBeInTheDocument();
     expect(getByText('Amount + fees')).toBeInTheDocument();
     expect(getByText('0.001000021000 ETH')).toBeInTheDocument();
+    expect(getByText('$0.56')).toBeInTheDocument();
+  });
+
+  it('should not contain a fiat value', () => {
```

Comment:
```suggestion
  it('should not contain a fiat value when balance and token price checker is disabled', () => {
```

optionally we can break the two new tests into more describe blocks

```
  describe('when balance and token price checker is disabled', () => {

    let storeWithPriceCheckerDisabled;

    beforeEach(() => {
      storeWithPriceCheckerDisabled = configureStore({
        ...mockState,
        metamask: {
          ...mockState.metamask,
          useCurrencyRateCheck: false,
        },
      });
    });

    // [optional after block] I added this here because, in the past, defining variables where we did in the describe block caused memory leaks in mocha. I haven't taken the time to test this in the latest version of jest, so I personally unset variables here just in case. 
    after(() => {
      storeWithPriceCheckerDisabled = null;
    });

    it('should match screenshot', () => { /** */ });
    it('should not contain a fiat value', () => { /** */ });
  });
```

Reply:
done